index.bs

```diff
@@ -322,21 +322,12 @@ they are also defined by how content uses those features.
 
 <h3 id="leave-the-web-better">Leave the web better than you found it</h3>
 
-As you add new capabilities to the web platform,
-do so in a way that improves the overall platform,
-for example its security, privacy or accessibility characteristics.
-
-The existence of a defect in one part of the platform
-must not be used to excuse an addition or extension to the defect,
-which would further decrease the overall platform quality.
-Where possible, build new web capabilities that improve the overall platform quality
-by mitigating existing defects.
-Do not degrade existing capabilities without good reason.
-
-Parts of the web platform evolve independently.
-Issues that are present with a certain web technology now may be fixed in a subsequent iteration.
-Duplicating these issues makes fixing them more difficult.
-By adhering to this principle we can make sure overall platform quality improves over time.
+As you add new capabilities to the web platform, do so in a way that improves the overall platform, for example its security, privacy or accessibility characteristics.
+The existence of a defect in one part of the platform must not be used to excuse an addition or extension to the defect, which would further decrease the overall platform quality. Consistency is not a good reason to propagate problems. Where possible, build new web capabilities that improve the overall platform quality by mitigating existing defects. Do not degrade existing capabilities without good reason.
+
+Do not extend a platform feature that has known defects, without considering how to address those defects. Extensions could create new ways in which the defect could cause problems, such as by encouraging more use of the defective feature. It is not necessary to completely fix all aspects of a defect, it is more important to consider mitigations or fixes when developing such an extension.
+
+Parts of the web platform evolve independently. Issues that are present with a certain web technology now may be fixed in a subsequent iteration. Duplicating these issues makes fixing them more difficult. By adhering to this principle we can make sure overall platform quality improves over time.
```

Comment:
This document tends to [use semantic line breaks](https://rhodesmill.org/brandon/2012/one-sentence-per-line/). I'm inclined to say that all patches should do that, but even if we decide not to, they should definitely wrap at 80-100 columns.

Also please avoid re-wrapping lines you're not otherwise touching, to make the diff easier to read.

Reply:
It looks like my IDE hard wrap setting didn't take effect. Thanks for catching this! I'll add the line breaks and push it back up.

Reply:
I think that you missed the intent of the comment.

When using semantic breaks, you attempt to keep one concept per line.  That means that lines end on periods and semi-colons always, and at the end of obvious clauses.  (This is not a good idea with GitHub markdown, which retains line breaks, BTW.)

That means not doing a hard wrap at any particular column (like your 100ch), but choosing where to break deliberately.